src/server/server.js

```diff
@@ -1,46 +1,51 @@
-import User from './src/models/user'
-import mailgun from 'mailgun-js' 
-import { generateDbConnectionFailedEmail } from './src/resources/emails'
-import path from 'path'
-import express from 'express'; 
-import cors from 'cors'
-import mongoose from 'mongoose'
-import dotenv from 'dotenv'
-import routers from './src/routes/'
+import User from "./src/models/user";
+import mailgun from "mailgun-js";
```

Comment:
add "mailgun" license into the README file

Reply:
done